reana_workflow_controller/utils.py

```diff
@@ -72,16 +72,20 @@ def remove_workflow_jobs_from_cache(workflow):
 
 
 def remove_files_recursive_wildcard(directory_path, path):
-    """Remove file(s) from workflow workspace.
-
-    :param directory_path: FIXME.
-    :param path: FIXME.
-    :return: FIXME.
+    """Remove file(s) fitting the wildcard from workflow workspace.
```

Comment:
Now we try to use only ``workspace``, not ``workflow workspace``. CLI e.g https://github.com/reanahub/reana-client/blob/256420656f8201ad5d81295c357b6cf35725b681/docs/cmd_list.txt#L36

Reply:
👍 fixed